packages/medusa/src/services/order.ts

```diff
@@ -1408,6 +1490,136 @@ class OrderService extends TransactionBaseService {
     })
   }
 
+  async decorateTotalsNew(
+    order: Order,
+    totalsConfig: TotalsConfig = {}
+  ): Promise<Order> {
+    const calculationContext = await this.totalsService_.getCalculationContext(
+      order
+    )
+
+    const includeTax = totalsConfig?.force_taxes || order.region.automatic_taxes
+    const useExistingTaxLines = totalsConfig.useExistingTaxLines
+
+    const [itemsTotals, shippingTotals] = await Promise.all([
+      await this.totalsNewService_.getLineItemsTotals(order.items, {
```

Comment:
I'm pretty sure if you add the await here, the Promise.all is unnecessary, since it's already awaiting when creating the array.

Reply:
Ahah dummy me, you are right